.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
             echo "2.3" > .ruby-version
             gem update --system
             brew install shellcheck
-            bundle check || bundle install --jobs=4 --retry=3 --path .bundle
+            bundle check --path .bundle || bundle install --jobs=4 --retry=3 --path .bundle
```

Comment:
meaning `bundle check` could never have worked with the `--path` used for `bundle install`?

Reply:
Yeah I don't think so. That's why it would seemingly always return "Bundler can't satisfy dependencies"

Reply:
Ok, Probably got broken in the transition from Circle CI 1.0 to 2.0. My guess is `--path .bundle` was added to the install but not the check.

Great you caught that!